src/Corax/Querying/IndexSearcher.cs

```diff
@@ -212,7 +212,9 @@ internal Slice EncodeAndApplyAnalyzer(in FieldMetadata binding, Analyzer analyze
     //Function used to generate Slice from query parameters.
     //We cannot dispose them before the whole query is executed because they are an integral part of IQueryMatch.
     //We know that the Slices are automatically disposed when the transaction is closed so we don't need to track them.
+#if !DEBUG
```

Comment:
The reason why you want it in debug is because if you dont do so, you can't debug this method as Visual Studio will trip into not initialized memory and kill the process. (It sucks big time).